routes/user.js

```diff
@@ -9,6 +9,48 @@ const Invitation = require("../models/Invitation");
 
 sgMail.setApiKey(process.env.EMAIL_KEY);
 
+//GET all user outgoing invitations
+router.get("/:id/invitations/out", async (req, res) => {
+  const userId = req.params.id;
+  try {
+    const invites = await Invitation.find({ referrer: userId });
+    console.log("invites:", invites);
+
+    if (invites.length < 1) {
+      res.status(404).send("No invitations found.");
```

Comment:
I'm not sure that Instead of 404, let's return a [204](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204) with an empty array, since there wasn't an error